hc/accounts/tests/test_login.py

```diff
@@ -16,22 +17,55 @@ def test_it_sends_link(self):
 
         form = {"email": "alice@example.org"}
 
-        r = self.client.post("/accounts/login/", form)
-        assert r.status_code == 302
+        response = self.client.post("/accounts/login/", form)
+        self.assertEqual(response.status_code, 302)
 
         ### Assert that a user was created
+        user = User.objects.filter(email="alice@example.org").first()
+        self.assertTrue(user)
 
         # And email sent
         self.assertEqual(len(mail.outbox), 1)
-        self.assertEqual(mail.outbox[0].subject, 'Log in to healthchecks.io')
-        ### Assert contents of the email body
 
+        ### Assert contents of the email body
+        self.assertEqual(mail.outbox[0].subject, 'Log in to healthchecks.io')
+        
         ### Assert that check is associated with the new user
+        check = Check.objects.filter(user=user.id).first()
+        self.assertTrue(check)
 
     def test_it_pops_bad_link_from_session(self):
+        """ Sets a bad_link session variable and verifies that logging in
+        removes it """
         self.client.session["bad_link"] = True
         self.client.get("/accounts/login/")
         assert "bad_link" not in self.client.session
```

Comment:
Assert format used is `self.assert` this does not follow convention. Please change it.